js/apps/admin-ui/src/identity-providers/add/DescriptorSettings.tsx

```diff
@@ -462,6 +432,45 @@ const Fields = ({ readOnly }: DescriptorSettingsProps) => {
         label="wantAssertionsEncrypted"
         isReadOnly={readOnly}
       />
+
+      {wantAssertionsEncrypted === "true" && (
+        <FormGroup
+          label={t("encryptionAlgorithm")}
+          labelIcon={
+            <HelpItem
+              helpText={th("encryptionAlgorithm")}
+              fieldLabelId="identity-provider:encryptionAlgorithm"
```

Comment:
We recently made some changes to the way labels are handled, so the namespace can now be omitted.
```suggestion
              fieldLabelId="encryptionAlgorithm"
```

Reply:
@jonkoops I had just moved it! :smile: Done now!

But one question, modifying this I saw that the help message was not OK. I needed to also change the `helpText={th("encryptionAlgorithmHelp")}` (adding suffix `Help`) in the `HelpItem`. I see a all the `HelpItem` (a least in the saml provider) are not showing the proper help message. I just modified the one that I was moving.

Reply:
Could you log a separate issue for that one? I think this might be because of the recent changes made to the labels. ping @edewit